typed_ast/ast3.py

```diff
@@ -48,8 +48,12 @@ def parse(source, filename='<unknown>', mode='exec', feature_version=LATEST_MINO
     Set feature_version to limit the syntax parsed to that minor version of
     Python 3.  For example, feature_version=5 will prevent new syntax features
     from Python 3.6+ from being used, such as fstrings.  Currently only
-    supported for Python 3.5+, so feature_version=4 or less are all equivalent
-    to feature_version=5.
+    fully supported for Python 3.5+ with partial support for Python 3.4.
+    So, feature_version=3 or less are all equivalent to feature_version=4.
+
+    If feature_version=4, the parser will currently forbid the use of the
+    async/await keywords and the '@' operator, but will not forbid the use of
+    PEP 448 additional unpacking generalizations, which was added in Python 3.5.
```

Comment:
Nit: I'd consider some minor wording tweaks here.  I think "When" is more commonly used than "If", in this context.  I think you can get rid of "currently" -- docs by default describe current behavior (and there aren't any concrete plans to change this).  Also, "additional unpacking generalizations" is plural, so I think the final "was" should be a "were".